ChemistryLib/PhreeqcInterface.h

```diff
@@ -0,0 +1,88 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2019, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#pragma once
+
+#include <memory>
+
+#include "BaseLib/ConfigTree.h"
+#include "Output.h"
+#include "PhreeqcInterfaceData/AqueousSolution.h"
+#include "PhreeqcInterfaceData/EquilibriumPhase.h"
+#include "PhreeqcInterfaceData/KineticReactant.h"
+#include "PhreeqcInterfaceData/ReactionRate.h"
+#include "ProcessLib/Process.h"
```

Comment:
There is still dependency of the ChemistryLib from ProcessLib.

Reply:
Removed and in the meanwhile add the header file `MathLib/LinAlg/GlobalMatrixVectorTypes.h` in the `PhreeqcInterface.h` and the header file `BaseLib/Algorithm.h` in the `PhreeqcInterface.cpp`.